airbyte-integrations/connectors/source-relational-db/src/main/java/io/airbyte/integrations/source/relationaldb/AbstractDbSource.java

```diff
@@ -144,7 +144,7 @@ public AutoCloseableIterator<AirbyteMessage> read(final JsonNode config,
 
     final Database database = createDatabase(config);
 
-    logPreSyncDebugData(database, catalog);
+    validateSyncSettings(database, catalog, state);
```

Comment:
The method `logPreSyncDebugData()` is now called from within `validateSyncSettings()`.